dependencies.yaml

```diff
@@ -117,13 +144,22 @@ dependencies:
               - python=3.10
   run:
     common:
-      - output_types: [conda, requirements]
+      - output_types: [conda, requirements, pyproject]
         packages:
           - numba>=0.49
           - numpy>=1.19
+  cuda-python:
```

Comment:
I think this _package_ should be listed in both build and run, rather than in its own _section_ that is duplicated in build and run?

Reply:
You are right, this is an artifact of an earlier iteration (while I was still developing the dfg-toml feature) when there were multiple extra sections and contents to centralize. A simple YAML alias will suffice now.